libs/platform/src/lib/components/panel/panel-actions/panel-actions.component.ts

```diff
@@ -3,6 +3,7 @@ import { Component, ChangeDetectionStrategy } from '@angular/core';
 @Component({
     selector: 'fdp-panel-actions',
     templateUrl: './panel-actions.component.html',
+    styles: [` :host ::ng-deep > * { margin-left: 0.5rem } `],
```

Comment:
this is not part of the Panel requirements. You don't need to address that. If you want to show actions in the panel header then you can wrap the buttons in overflow toolbar - https://fundamental-ngx.netlify.app/#/core/toolbar#Toolbar-overflow

Reply:
@droshev as I see it's already wrapped in the overflow toolbar
https://github.com/SAP/fundamental-ngx/blob/ed2f058658891fcc230c4163bc8aac4540266d7b/libs/platform/src/lib/components/panel/panel.component.html#L12

![image](https://user-images.githubusercontent.com/14160094/92571752-2ba81380-f28c-11ea-8eaf-d2be27dc0f50.png)